### PR TITLE
feat: unsupported browser fallback

### PR DESCRIPTION
**Pull request by r0skar**
- Moved our unsupported browser page to a subfolder in the public dir
- Updated to check for [flex-gap support](https://github.com/w3c/csswg-drafts/issues/3559#issuecomment-1758459996)

### CODE REVIEW

File: public/unsupported-browser/browser-check.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function browserCheck() {
```
Comment (edoardolunardi) [inline public/unsupported-browser/browser-check.js:1]: Very minor but I would call this function `featureCheck`

File: public/unsupported-browser/browser-check.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  // https://github.com/w3c/csswg-drafts/issues/3559#issuecomment-1758459996
+  const supportsFlexGap = typeof CSS !== "undefined" ? CSS.supports("inset: 0") : false;
```
Comment (edoardolunardi) [inline public/unsupported-browser/browser-check.js:3]: Since this can run on old browsers, to be safe, maybe better to use `var` instead of `const`?

**Review by edoardolunardi — APPROVED** [review]
lgtm, 2 minor comments

File: public/unsupported-browser/browser-check.js (hunk 2 above)
Comment (r0skar) [reply public/unsupported-browser/browser-check.js]: haha yeah i guess we should keep IE10 in mind :) I changed it :+1:

**Pull request merged**